### ISSUE TITLE
Polygon tool

### ISSUE THREAD

**Issue by dsizzle** [HaikuArchives/ArtPaint#343]
Separate from what was discussed with the polyline tool, there ought to be a tool to draw perfectly regular polygons with a selected number of sides.  Bonus points if the sides can be made concave, to support star shapes (e.g. 5-sided polygon is a pentagon, but with concave/indented sides becomes a 5-pointed star).

**Comment by humdingerb** [comment]
While it would be nice, be careful not to stray too far into vector app territory... :)

**Comment by dsizzle** [comment]
True. Well, I’m just jotting ideas. Probably wouldn’t get to this for quite some time yet.
